controller/src/main/java/org/jboss/as/controller/RunningModeControl.java

```diff
@@ -32,6 +32,7 @@
     private volatile RunningMode runningMode;
     private volatile boolean reloaded;
     private volatile boolean useCurrentConfig;
+    private String newBootFileName;
```

Comment:
volatile?

I haven't really looked into whether that's needed; I just see that other stuff is volatile.